packages/data-service/src/connection-storage.ts

```diff
@@ -69,7 +69,7 @@ export class ConnectionStorage {
   }
 }
 
-function promisifyAmpersandMethod<T>(
+export function promisifyAmpersandMethod<T>(
```

Comment:
@gribnoysup this thing is so unrelated, where could we extract it so it can be used in different places?

Reply:
Tbh I can only 🤷 I don't think we have a place for something like that. I am thinking more and more of moving all ampersand models in one package, then it would kinda makes sense to have it there? What do you think?

Reply:
Yeah, that makes sense, definitely would belong there.